lib/NativeCall.pm6

```diff
@@ -403,7 +390,7 @@ sub cglobal($libname, $symbol, $target-type) is export is rw {
     Proxy.new(
         FETCH => -> $ {
             nqp::nativecallglobal(
-                nqp::unbox_s(guess_library_name($libname)),
+                nqp::unbox_s(guess_library_name($libname).IO.absolute),
```

Comment:
This worries me, `guess_library_name` can properly return `''` when we request a symbol for an already loaded library.